plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -78,42 +56,46 @@ protected static function getTextBasedOnStock( $is_in_stock, $is_low_stock, $low
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
-		if ( ! empty( $content ) ) {
-			parent::register_block_type_assets();
-			$this->register_chunk_translations( [ $this->block_name ] );
-			return $content;
-		}
```

Comment:
Not a big deal, but I think we should keep this. It was preventing all the logic to run unnecessarily in the _All Products_ block.

Reply:
Done in [aefebbe](https://github.com/woocommerce/woocommerce/pull/53462/commits/aefebbeed7936ec68f0ebc583ed279892d893e44).